.circleci/ansible/scripts/init_safekeeper.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+
+# get instance id from meta-data service
+INSTANCE_ID=$(curl -s http://169.254.169.254/latest/meta-data/instance-id)
```

Comment:
I wonder why we don't use some domain name here as well

Reply:
I guess we use instance_id as something that is fixed and unique, this way it's better than domain names.

Reply:
Yeah, but I mean this IP 169.254.169.254

Reply:
It's a special AWS IP for obtaining metadata, I don't see that it has domain name https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instancedata-data-retrieval.html